config/middleware.rb

```diff
@@ -0,0 +1,5 @@
+require 'request_headers'
```

Comment:
This is file is obsolete

Reply:
Ok, removed

Reply:
Hey Aleix, to me it seems this file is still here